src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,40 +2,37 @@
 
 import stack.IntLinkedList;
 
-public class DefaultCountingOutRhymer {
-
-	private final IntLinkedList intLinkedList;
 
-	public DefaultCountingOutRhymer(IntLinkedList intLinkedList) {
-		this.intLinkedList = intLinkedList;
-	}
+public class DefaultCountingOutRhymer {
+	
+	protected Stack list;

```

Comment:
"stack" rather than "list".
There should be a constructor: public DefaultCountingOutRhymer(Stack stack)